src/wristpy/core/cli.py

```diff
@@ -0,0 +1,100 @@
+"""CLI for wristpy."""
+
+import argparse
+import pathlib
+from typing import List, Optional
+
+from wristpy.core import config, orchestrator
+
+logger = config.get_logger()
+
+
+def parse_arguments(args: Optional[List[str]] = None) -> argparse.Namespace:
+    """Argument parser for Wristpy's cli.
+
+    Args:
+        args: A list of command line arguments given as strings. If None, the parser
+            will take the args from `sys.argv`.
+
+    Returns:
+        Namespace object with all of the input arguments and default values.
+    """
+    default_settings = config.Settings()
+    parser = argparse.ArgumentParser(
+        description="Run the main wristpy pipeline",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+        epilog="Please report issues at https://github.com/childmindresearch/wristpy",
```

Comment:
I don't see the epilog in the help copy you posted.

Reply:
My fault, I cut it off by mistake: 

```
usage: wristpy [-h] [-o OUTPUT] [-c {ggir,gradient,none}] [-t THRESHOLDS THRESHOLDS THRESHOLDS] [-e EPOCH_LENGTH] [-v] input

Run the main wristpy pipeline

positional arguments:
  input                 Path to the input data.

options:
  -h, --help            show this help message and exit
  -o OUTPUT, --output OUTPUT
                        Path where data will be saved. Supports .csv and .parquet formats. (default: None)
  -c {ggir,gradient,none}, --calibrator {ggir,gradient,none}
                        Pick which calibrator to use. Can be 'ggir' or 'gradient'. (default: none)
  -t THRESHOLDS THRESHOLDS THRESHOLDS, --thresholds THRESHOLDS THRESHOLDS THRESHOLDS
                        Provide three thresholds for light, moderate, and vigorous activity.Values must be given in ascending order. (default: [0.03, 0.1, 0.3])
  -e EPOCH_LENGTH, --epoch_length EPOCH_LENGTH
                        Specify the sampling rate in seconds for all metrics. Set to 0 toskip downsampling. (default: 5)
  -v, --version         show program's version number and exit

Please report issues at https://github.com/childmindresearch/wristpy
```